sdk/identity/Azure.Identity/src/KnownAuthorityHosts.cs

```diff
@@ -10,24 +10,50 @@ namespace Azure.Identity
     /// </summary>
     public static class KnownAuthorityHosts
     {
+        private const string AzureCloudHostUrl = "https://login.microsoftonline.com/";
+        private const string AzureChinaCloudHostUrl = "https://login.chinacloudapi.cn/";
+        private const string AzureGermanCloudHostUrl = "https://login.microsoftonline.de/";
+        private const string AzureUSGovernmentHostUrl = "https://login.microsoftonline.us/";
         /// <summary>
         /// The host of the Azure Active Directory authority for tenants in the Azure Public Cloud.
         /// </summary>
-        public static readonly Uri AzureCloud = new Uri("https://login.microsoftonline.com/");
+        public static readonly Uri AzureCloud = new Uri(AzureCloudHostUrl);
 
         /// <summary>
         /// The host of the Azure Active Directory authority for tenants in the Azure China Cloud.
         /// </summary>
-        public static readonly Uri AzureChinaCloud = new Uri("https://login.chinacloudapi.cn/");
+        public static readonly Uri AzureChinaCloud = new Uri(AzureChinaCloudHostUrl);
 
         /// <summary>
         /// The host of the Azure Active Directory authority for tenants in the Azure German Cloud.
         /// </summary>
-        public static readonly Uri AzureGermanCloud = new Uri("https://login.microsoftonline.de/");
+        public static readonly Uri AzureGermanCloud = new Uri(AzureGermanCloudHostUrl);
 
         /// <summary>
         /// The host of the Azure Active Directory authority for tenants in the Azure US Government Cloud.
         /// </summary>
-        public static readonly Uri AzureUSGovernment = new Uri("https://login.microsoftonline.us/");
+        public static readonly Uri AzureUSGovernment = new Uri(AzureUSGovernmentHostUrl);
+
+        internal static string GetDefaultScope(Uri authorityHost)
+        {
+            switch (authorityHost.ToString())
+            {
+                case AzureCloudHostUrl:
+                    return "https://management.core.windows.net//.default";
+                case AzureChinaCloudHostUrl:
+                    return "https://management.core.chinacloudapi.cn//.default";
+                case AzureGermanCloudHostUrl:
+                    return "https://management.core.cloudapi.de//.default";
+                case AzureUSGovernmentHostUrl:
+                    return "https://management.core.usgovcloudapi.net//.default";
+                default:
+                    return null;
```

Comment:
I wonder if we should also support an optional .json config file in the future that lets you customize these mappings when placed next to your assembly or something.

Reply:
Long term there will need to be more advanced configuration for different clouds, especially to fully support Azure Stack. However, the management scope for all these clouds cannot change.